docs/docs/external/getting-started/interop/data-sharing.md

```diff
@@ -1,6 +1,7 @@
 ---
 id: data-sharing
 title: Data Sharing
+pagination_next: external/getting-started/enabling-weaver-network/overview
```

Comment:
`pagination_prev: external/getting-started/interop/overview`
This can be added here also? like other two pages.

Reply:
Yes, this was an omission. Added now.